exonum-java-binding/core/rust/ejb-app/start_cryptocurrency_node.sh

```diff
@@ -65,6 +68,8 @@ cargo run -- finalize testnet/sec.toml testnet/node.toml \
 
 header "START TESTNET"
 cargo run -- run -d testnet/db -c testnet/node.toml \
+ --consensus-key-pass pass \
```

Comment:
Is that needed although you previously pass `--no-password`? Does it mean 'pass the password check' or is it a password that is allowed to be entered in CLI?

Reply:
🤷‍♂️ This means no password. Password is specified as `pass:{password}`

Reply:
I was surprised by this, @dip56 provided an explanation

Reply:
> 🤷‍♂️ This means no password.